docs/resource/create_new_resource.md

```diff
@@ -126,10 +126,8 @@ class BookRepository extends ServiceEntityRepository
 ```
 {% endcode %}
 
-The generated code is not compatible with Sylius Resource yet, so we need to make a few changes.
-
-* First, your repository should implement the `Sylius\Component\Resource\Repository\RepositoryInterface` interface
-* Then, add the `Sylius\Bundle\ResourceBundle\Doctrine\ORM\ResourceRepositoryTrait` trait
+The generated code is not fully compatible with Sylius Resource yet, so we need to make a few change.
```

Comment:
This is merged already but just noticed here a typo was introduced so will need fixing 
```suggestion
The generated code is not fully compatible with Sylius Resource yet, so we need to make a few changes.
```